app/scripts/modules/kubernetes/src/rawResource/component/group/RawResource.less

```diff
@@ -0,0 +1,81 @@
+.raw-resource-card {
```

Comment:
please scope all CSS using a selector with the same name as the React component it is intended for, i.e.:

```less
.RawResource {
  .card {
    ...
  }

 .column {
   ...
 }

 ...
}